modules/framework/galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/spi/teststructure/TestStructure.java

```diff
@@ -203,6 +204,17 @@ public String getRunName() {
         }
         return this.runName;
     }
+
+    public void setGroup(String group) {
+        this.group = group;
+    }
+
+    public String getGroup() {
+        if (this.group == null) {
+            return "none";
+        }
+        return this.group;
```

Comment:
Instead of multiple returns, we could store the group name in a variable and return that at the end?